### PR TITLE
add snippets support

### PR DESCRIPTION
**Pull request by MortalCatalyst**
Just wanted to add snippets support for jinja2 to make life a little easier.

### CODE REVIEW

**Comment by MortalCatalyst** [comment]
Note I may have butchered the package json as haven't authored vscode extensions before.

**Comment by wholroyd** [comment]
No worries, everyone has to start somewhere. I have an update coming soon and I'll see if I can incorporate it.

**Comment by xen** [comment]
Also, I found very handy to have support for babel translation. So, please also include this snippet:

```json
	"i18n": {
		"prefix": "_",
		"body": [
			"{{ _('${TM_SELECTED_TEXT:${1:text}}') }}"
		],
		"description": "i18n"
	},
```

**Comment by xen** [comment]
I updated and extended the list of snippets. If you like you can use my gist https://gist.github.com/xen/d11451454820554cb61f20dcdad3a4b0

File: package.json [hunk 1]
```diff
@@ -68,7 +68,14 @@
 				"language": "yaml",
 				"scopeName": "text.yaml.jinja",
 				"path": "./syntaxes/jinja-yaml.json"
-			}
+			},
+                    "sniipets": [
```
Comment (jonathan-s) [inline package.json:72]: Typo here, should be `snippets`, right.

**Comment by jonathan-s** [comment]
@wholroyd Is there anything I could help with to move this PR into being merged?

**Comment by wholroyd** [comment]
Pull the changes and incorporated them in https://github.com/wholroyd/vscode-jinja/pull/23.

Thank you for the contribution!

Not going to close this just yet, just saw @xen 's Gist.

**Comment by wholroyd** [comment]
Added in @xen 's snippets for i18n also into #23.

**Pull request closed without merging**